monkey/monkey_island/cc/next_ui/src/app/(protected)/plugins/_lib/filters/InstalledPluginFilter.tsx

```diff
@@ -0,0 +1,26 @@
+import { PluginRow } from '@/app/(protected)/plugins/_lib/PluginTable';
+import { useGetInstalledPluginsQuery } from '@/redux/features/api/agentPlugins/agentPluginEndpoints';
+import { FilterProps } from '@/app/(protected)/plugins/available/AvailablePluginFilters';
+import { useEffect } from 'react';
+
+type InstalledPluginFilterProps = FilterProps;
+
+const InstalledPluginFilter = (props: InstalledPluginFilterProps) => {
```

Comment:
It's called hooks. We could refactor this to `useInstallPluginFilter` hook, but as it stands now this component is consistent with the other filters and might need a display component in the future (it could be a toggle on/off to show installed plugins for example). AFAIK components that don't render DOM is not an anti-pattern.